### PR TITLE
Blog post for HSearch 5.5.6.Final

### PR DESCRIPTION
**Pull request by yrodiere**
http://staging.in.relation.to/2017/01/09/hibernate-search-5-5-6-Final/

### CODE REVIEW

File: posts/Yoann/2017-01-09-hibernate-search-5-5-6-Final.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+== What's new?
+
+ * https://hibernate.atlassian.net/browse/HSEARCH-2494[HSEARCH-2494]: `@TikaBridge` will now work correctly on properties of type `byte[]`.
+ * https://hibernate.atlassian.net/browse/HSEARCH-2535[HSEARCH-2535]: `@Facet` with string encoding will now work properly on multi-valued properties (such as `String[]` or `List<String>`).
+ * https://hibernate.atlassian.net/browse/HSEARCH-2486[HSEARCH-2486]: `@ContainedIn` in a superclass will now be taken into account even if the concrete class does not carry any Hibernate Search annotation.
+ * https://hibernate.atlassian.net/browse/HSEARCH-2479[HSEARCH-2479]: building phrase queries with the Hibernate Search DSL used to trigger an `IllegalArgumentException` in some specific cases; this has been fixed.
+ * ... and more: the full change log can be found on https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10061&version=25505[our JIRA instance].
+ 
+Thanks to Julien Benichou, Andrew Robie and Timo Tretted for reporting issues, and even fixing one!
```
Comment (gsmet) [inline posts/Yoann/2017-01-09-hibernate-search-5-5-6-Final.adoc]: s/Tretted/Tretter/ IIRC

Comment (gsmet) [reply posts/Yoann/2017-01-09-hibernate-search-5-5-6-Final.adoc]: And you can probably put an accent on Bénichou.

Comment (yrodiere) [reply posts/Yoann/2017-01-09-hibernate-search-5-5-6-Final.adoc]: Right, that's a bad one. Thanks.

File: posts/Yoann/2017-01-09-hibernate-search-5-5-6-Final.adoc [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+== How to get this release
+
+All versions are available on Hibernate Search's http://hibernate.org/search/[web site].
```
Comment (gsmet) [inline posts/Yoann/2017-01-09-hibernate-search-5-5-6-Final.adoc:23]: I would skip the `'s` in this context.

**Pull request merged**